apps/examples/st_things/sample_light_user.c

```diff
@@ -0,0 +1,93 @@
+/* ****************************************************************
+*
+* Copyright 2017 Samsung Electronics All Rights Reserved.
+*
+*
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+******************************************************************/
+
+#include <tinyara/config.h>
+
+#include <stdio.h>
+#include <string.h>
+#include <stdbool.h>
+#include <inttypes.h>
+#include <signal.h>
+#include "st_things/st_things.h"
+#include <tinyara/gpio.h>
+#include <fcntl.h>
+
+#define TAG                     "LIGHT_USER"
+#define BLUE_LED 49
+
+static const char *PROPERTY_VALUE = "value";
+
+bool g_switch_value = true;
+
+static void gpio_write(int port, int value)
```

Comment:
We provide iotbus_gpio. Can't you use that?

Reply:
@KimDongEon Please review

Reply:
This is also defined by IoT lab. I will check that whether it could be changed or not.

Reply:
can use iotbus_gpio.
I changed it to iotbus_gpio.